task/build-vm-image/0.1/build-vm-image.yaml

```
@@ -148,6 +154,22 @@ spec:
         echo "$BUILD_DIR"
         ssh -v $SSH_ARGS "$SSH_HOST"  mkdir -p "$BUILD_DIR/workspaces" "$BUILD_DIR/scripts" "$BUILD_DIR/tmp" "$BUILD_DIR/tekton-results" "$BUILD_DIR/entitlement"
 
+        if [! -n "${CONFIG_TOML_FILE}" ]; then
```

```suggestion
        if [ ! -n "${CONFIG_TOML_FILE}" ]; then
```
otherwise, I think this is a `[!: command not found`